### ISSUE TITLE
Build fails on Windows

### ISSUE THREAD

**Issue by FSM1** [UniversalLogin/UniversalLoginSDK#152]
The build command fails when executed on Windows.

Resolution is probably to integrate cross-env on the babel calls.

**Comment by yaram** [comment]
I've had the build fail on windows as well.

All the errors are of a similar form to this:
```
| contracts\ENSRegistered.sol:3:1: ParserError: Source "universal-login-contracts\ENS\ENS.sol" not found: File not found: ENS/ENS.sol
 | import "universal-login-contracts\ENS\ENS.sol";
```

I belive the issue is that Waffle is basing the import off of the current working directory, which is "universal-login-contracts".

The place where the files are searched for and the error is printed is [here](https://github.com/EthWorks/Waffle/blob/c7de9c7fd165380a965bf7928c8562f1e0857e62/lib/compiler.ts#L44).

**Comment by yaram** [comment]
As for why the working directory is different on Windows, I'll try figuring that out.

**Comment by yaram** [comment]
It seems that the function in Waffle isn't even called on Linux. I believe the source of the error is then solcjs or the arguments/parameters passed to it.

**Comment by yaram** [comment]
Through testing I've found that solc is run with the same working directory on windows and linux, so the working directory is not the difference.

**Comment by marekkirejczyk** [comment]
We don't have plans to support Windows as a development environment. Only Linux and OSX. If you make it work, however, I'll be happy to accept Pull Request.

**Comment by yaram** [comment]
Just an update for anyone interested, I've fixed this in EthWorks/Waffle#59
